androidsdk/src/main/java/com/envoy/androidsdk/domain/model/CreateLinkBody.kt

```diff
@@ -3,40 +3,64 @@ package com.envoy.androidsdk.domain.model
 import com.google.gson.annotations.SerializedName
 
 data class CreateLinkBody(
```

Comment:
What using "java doc" style and having the comments above the class. Same for all data classes